assets/sass/utilities/_utilities-accents.scss

```diff
@@ -27,108 +22,3 @@
 .u-border-top-brand-primary {
     border-top: 4px solid get-color('brand', 'primary');
 }
-
-.u-border-minor {
```

Comment:
This is gone, I replaced the one use case with a `<hr/>` which is a semantically correct thing to use here so we can avoid the extra styles.